lefthook.yml

```diff
@@ -25,7 +25,11 @@ pre-commit:
       exclude:
         - __fixtures__/*
 
-    - run: yarn prettier --log-level=silent --check {staged_files}
+    - name: prettier staged files
+      run: yarn prettier --log-level=silent --check {staged_files}
+      glob: '*.{js,mjs,ts,mts,jsx,tsx}'
```

Comment:
fixes this pre-commit error:
<img width="847" height="101" alt="image" src="https://github.com/user-attachments/assets/56cc468e-9cda-46b7-870b-11e82249a408" />